tests/run-make/apple-slow-tls/rmake.rs

```diff
@@ -0,0 +1,28 @@
+//! Test if compilation with has-thread-local=false works, and if the output
+//! has indeed no fast TLS variables.
+
+//@ only-apple
+
+use run_make_support::run::cmd;
+use run_make_support::{cargo, rfs, rustc};
+
+fn main() {
+    let output =
+        rustc().print("target-spec-json").args(["-Z", "unstable-options"]).run().stdout_utf8();
+    let to_search = r#""has-thread-local": true"#;
+    assert!(output.contains(to_search));
+    let output = output.replace(to_search, r#""has-thread-local": false"#);
```

Comment:
Nit: I believe [`run_make_support` does re-export `serde_json`](https://doc.rust-lang.org/nightly/nightly-rustc/run_make_support/index.html), consider using structured JSON manipulation instead of plain-text replacement.

Reply:
I was expecting it also reexports `serde`, but it didn't :(